aws/cloudformation/components/alarms.yml.erb

```diff
@@ -26,8 +26,8 @@
       Dimensions:
         - {Name: InstanceId, Value: !Ref <%=daemon%>}
       EvaluationPeriods: 7
-      MetricName: MemoryUtilization
-      Namespace: 'System/Linux'
+      MetricName: mem_used_percent
+      Namespace: CWAgent
```

Comment:
We might want to change the `TreatMissingData` property below to `breaching` so that we act on missing data with more urgency https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-cloudwatch-alarm.html#cfn-cloudwatch-alarm-treatmissingdata

Reply:
That seems reasonable; I'll do so in a follow-up PR, though, since that's unrelated to the metrics switch in this PR